src/components/timeSlotBooking/timeSlotBookingSteps/TimeSlotFinalizeStep.tsx

```diff
@@ -277,12 +277,12 @@ export default function TimeSlotFinalizeStep({
             label="proposal-booking-finalization-strategy"
             options={[
               {
-                key: ProposalBookingFinalizeAction.CLOSE,
-                label: 'Close proposal booking',
+                key: ProposalBookingFinalizeAction.COMPLETE,
+                label: 'Complete time slot booking',
               },
               {
                 key: ProposalBookingFinalizeAction.RESTART,
-                label: 'Restart the booking process',
+                label: 'Restart time slot booking process',
```

Comment:
I think we can preserve the definite pronoun here 'Restart time slot booking process'

Reply:
@jekabs-karklins I don't really understand what is your point here. Can you explain a bit better? Do you mean we should keep it something like: `Restart the time slot booking process`?

Reply:
Yes :)